### ISSUE TITLE
Unlimited air units storage exploit

### ISSUE THREAD

**Issue by Anon177013** [yairm210/Unciv#11901]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Game Version

4.12.6-patch1

### Describe the bug

You can bypass the 6 air units per city limit by putting a carrying unit in the city, allowing you to stack infinite air units.

### Steps to Reproduce

Using the attached save game (map edited for testing this) :

If you move the carrying unit out of a city then press next turn, the city will stop producing the air unit, displaying "No space to place this unit" in the city screen. But if you keep the carrying unit in the city, it will keep producing air units forever, with no limit.

### Screenshots

![Screenshot_20240701-233800](https://github.com/yairm210/Unciv/assets/146986415/ce8cb88b-3a8c-4c67-8f8e-ab0dd5efe9cd)

### Link to save file

<details>
  <summary>Show Saved Game</summary>
  
```
H4sIAAAAAAAAAO19e4/bOLbnV9EaWKAHW+4RJerVuLhApborydxUUpOq2ewiyB+yrSrrxpY8kpzHBPXdl4cPSZT1YDmyTfcOZrq7JEs/HZ4HeXjOIfljMgsz8v84TPLJbz+eLibz+Eu8iv8VFnEKtz7+mMzD+TJa3IQF+U9+Ha+KKHsf5dtVQV6YfAjJ5eS3h3CVRxeTF2Hyubx4Fa9W1QX5TFo9eJ1F+dL4Kr38OkuT8uKuSJOovHqZRWFhvIySKAsrzKs0zIv6t8n/8vL6Jt0mRRhXkFdhUawqzHfzKKx+vI7zZe3jGaHsMZ4bhMx0m82jOl3zz8Yf3+bLMHmMaqRss2KZbvPq1k2cVMTdbWdrwuVak25XhLaK2vssXMTJo7FJa026IUyrUUwuP0fVr79HNebdLSX0+2i9qb2ZpMn0oYXlf9smj7XHXqVZHuU1zCjaVFdRuEmz+ufzqHb5YUlExK+oGhVxxPUnLr5fvr5O51tyPbnN0sV2DtrF7tBHv18RZSsydh8UcTLbEkV7Qf4FXAEcIs9ku44IT0EIhFnbNfnrcr1ZxgV8mrSK6x+RRpwxaK51RLzhvEiz7/BXmhG5XUzexI9LLjEQ3yrN84hCvo/m3+crkMUV+RqFvfznNgKiyZ830SKehyvjTTiDqzBZEq0KE4M0678j+sSLdD0zCOtW7N3b7WxFFOluvkzTFWARFich/HVL4MNHovOEeWn2OV+mGyA5zTdxQR+4IYZYhNl340VIqXyZEQWnjWhoIaCuWfP+K4uoCl1M3lITJqTeE+PJt+y9TTgnWhBvjIohL7I0XMyJJRn36VdGcrgijwEt2xnRP4O+ZdxHqyifp5uojv3HJp7D9Xa+isLMIEotBASKW2vC5TxcROvvk1KLiQFuiDUbr7gI/pHEX6IsJ7pALu5BohmVzNuSrFdRlhJO8k+SLoh8cb4Ukthy8V2RltBPC30FUc8yxrcP4WqVTz4Rnavp29+30TYCDftbVBjXoBb0c6e8Agq3WUYUpG4Yr/N/EJO7gy6A3CI2R0w6Km2EaOCXeBEtrrN0fb+M8ytgJbGk0H4wbcd6mKIwnE1x4FjTEEez6Yz8y1741nxuRcxOV2EOFvrf4ZfwVyLIx19JP0iQCW1fwhWwqEV/ni6UXmwq2dMn0kdsyq7gHRH9wyr9OvktwE9UOgkxpmhxn16F1BqIQhW0+4CGE1yiS2FCGvu+7J8nr5N5lFBViogFEe2E7hlGtOjbhoxuMQwuhFYybm2ziBhQFi0mv1luQJ54IEMF8PAq/kKA3m/zPA4JDlHCVUFGBdtyLiYxeVqNl+TZ/F0WP8bEQDj5QmCrdE4tByj5NvkNX0yIiKY2ISEJ19CIm5SY2FcCsUk321X57EOaLkqSLVf+Ga7zTTSPw1WcF5cr8RHaj/6REDroQGE/kc9k0YoQRkHJVRGveB8tEfOj+oPcd9l9LF9a/NJu/RXxS47pyg833hUPW/LD1oVERkmX+N2VPilAkPwWkjEdmVpPftiRyXNk8ji1jkyeJ8jyxFvS2w2yPJkv4ldf/rDbaLncBtzKxEYbHPlhLBNtyZfiVyza4os/sERgozGW9A3x4xTJoOLal19uUIRkiTT0p8FzR266JzfdlqEs0RbxlicT4kiPoyfS9X4lozHpe+rG0VBz3HrptsvAbeHyrmqiVsgGIzqEvo9dNQRQ6k8751GvIPrtqDScT3SwOLo32OIEwlQDpJz3OYJ33xdJ9N2Q3ZTneIfKnuCHOFmsYbZEnKJvD2m2MCRv6BDeoZoneJeual4dc6uNm/BbvKay6nPzSheRsHQVUTKe5/kdwMF7uQUHybiJyQBPITS9Mab7N587QRA6aLrw5+YUR+RfoR8+TD1sYs+3PBeHpqr7p+zFIcs7qhtnm/Zz3Dglnux4a6xbcypn7a741biNiHHns2322Ou0BYH8K8L9zhjrM2VPRHYe5L5fHjZkf0seB+RhQPaTZJdPdkdkD8+Xen4xDvjSF9iTgdQGLF35EtWeRLXsR1nS91zpe570Pfm9QGotlr5gS1+XRyvZ2ZKplsdE2SmypCt4TyJAdryQRLjMfLmJglGB1ChPar7fwm5b+p48tsvNkIdyX8KUWepKT8pq4Um/BeBMETNNLuf/3MbMUFu9K1kCbousZEfUbWWcbDO4BUV2DV0F5vgtzJG1AZ/KrdkNcr2KsjXpZelYW43q3f4NhFiND9wNOrRjczon5nDBqQO4KILYyjfQ/c6ozgrG5sMDmk9DL7Sn2A3QNMCIjM7WPJyZ0WwRLmY/5azECfn0Yxbl+Y4xNtv1m+WZT8cNRvnPcmJUeLXjxEzLvlZ4MW/TL48p6a96/RcU4EbQyRwl6DSVnZlpYyY7lT2RqRzrmcpjw1QeAKay9zOVHZ5pI1I1rU+aJYrKF+SAhi1fug3ckopGFGkq+zPlFHoq+0dTp62ljYerjwSNj8gD6lQeC6e4DduW+WTJl413m19qCqwEEyGl8n1fZlsjatT4auPXCkQO7VRviT+8BlWeDNTgoy2/JlTAkdvoyLQ5jW8h+dra9X38Vt9ngP2Nltttv1bM71DRhuo35O/JIK2qZqkx0GlDruk+6YM/idzgFeR0uU84j7+0O0evi2idfyDOD+lQIY0FHSPNkP6ovAgAqPsn4NCJdLNdy1va5dAPf+6mx8ATLh0WAGnmeu3S6YJH+WgPd4X3QXElb4PC7Hgp9B2ahwSgepCFXnPHT6KSZr3g1yp0BVdleoxeyPEZuFW5hHBVZhzpZ3bDW3C75uoA8TthHIranheEn3bcSGgFd92oYGpJPHh+N95Fedv0zACFOU60ocy/YtDM16RKIDmTcEdyVOGG7KrCHcmHoyg7OV34ZD07B9eNiA3c2nEg0FPD7Rk7HVc66HQGMvx8XdbUEEeMDf00LSO6fj9HC5faHXg3D/F8x2kFKZZfUKRISeqK1CpJTRVLhesVS4jj28qOK+bI/v/CDVYm9Zb5z6WLXnr6pAch/85Kh38FPvT7iLvAk1laFOn6PXQmwilHJndYTHCQ082b6EH8xn+y6FzkMV2BM+65jskuouTyMaL9SrJdz6Ls3cPL2l1IDFMH/lW42RCPHNqPTAszzySHr5Bh6IrN3soX2ezgEYbn2yilZUQ/xGWWp8ltGicFmT9tWf/+g1dn1Zx+mCywmzdRBqMn9KGuaYu7l1kRwyDuOuWtu3lMaKB3A0jJb+hHCHVvo28FfYRXf+18t6wKs0yz9UVGde098mFsUn4uw/wPMu68+5pEi66UPRlr7qIoeb3eEIWPaHCGYPxiX0zxX4j4r8I1xDR+wRc2XF6HGUQAfjEvpvT6BqqyLujjVv0a899h5mvMRcTlF+J90cduyVe3mXgT1d+0LqZu4xHi4NmNR6RrgorqxBEvfSrdsC9w/XmXX4bJ9qEMrACK2UAx2x5DMjGI01t9y2rQJnOGOJFW/Xmn8bvDaX9D5tzrMPvM30ENbjuUu1ui/+vcEOEv+Br9Acr9wEl5kYZFzlBNmUinLtyp02DYDo2UYe/ilfE1ouGuX7wLt/VDZK7GoZq/YFkIFlewGqMkDSNuvNRm3NAT8rvVomEOZ03L56c7jcCc99Q5ZTN/xl1ZvSxGuSQRq8khp4UY96JhJHajBTLXeYtkashTkuhMmY1EHSVCCGGyLSD2vLhGnA31xvAvCEhi234rCy3505i1uSZQ6ZpMjyhl7x4e8iXppWEKUBBveM0edhpG7jTEj3aeIICMX3/fhhkzRtRpCF7drGg0iMCEqw9psiD9Jc2WZoQFpAfPQ4jxJWkB3pBUF1xEj9SvB7FCtScZV0inS64/iqDSJ4N0sQYNeoG7HBe5QeZl8IX/QZ6O5wwLpjzFa7j4q/AkyBfDefWtNzy8dDkX0almXKuMCTzBYD/8vAiRPLE5qdS8N+kj/erz2sxqt47W4rKkSK3BZXnXE09YaCU95yekR17ghF8aH+sOBiN/RgZvIDsx4sT4KOdsP0ntkJyT5zZB5Aa5PoGDxfPgz9WjBonHkoHIH6pJoKO5/vObe2SzETG+P4PVlDHr/fo8LjQbnY2OeiPoqG1pr6PWsbr2xyz9mnQQf1tle56tmB2K5jyf8ydwJH7SmtyzsSZ3DGvaY4A7sjXZP2FNtYYG7Q3ljfoQGW/SL5Fxv4yM/4JJwe/hd+pwNBQY2vprvX21uAJvZm2K9BOGR4gFpOrROc+xd7tPNVpZSjs3PvJEtuwo9dJMQ+o0D85+E6nwn2sL1BZ+iaOv5eOydLDZK50yyJ8+GB+vt6BGxn00XzKhzFNYLFdEiw7Fo7Gp+U4LAKCiviBXPCJY+8BES78hGMVvwOfjNzgj9HS4w29ozj14PLJt8sE7vpZJhwhiPrdhPUP/P5K4qLH+x8uQzPQXxkf0aYfSLXn0l/wvTz+Mpx+LbWQUqREu0k0B/djHD2GWxWlmXKWL6NOTMuk34QYoUKack/l/0y1hHOlIaWdC6A0NFpg2Lh+jDq2ootsNGq5gLXO6iRZxWKMjhjXKk/ttkWZxuPprBT9pCNzWfmh7ZtDjOY5ivSUHchM7RlqMB/le1VfWKK6laOvGB2UEdevraEYdsbEUc0+1vuhXQID5a/1Lu0JpbaaceW5taaOXVWjx7gKE5zS66jBV2tz4lmKzG9n11naXI6lCi1vKGJ/T5KanNdDm5tcaGu/AxWYVfo+y++9QSTF5tSUO2ATSWat4HrMUH+2To8VteevjpN45T/guAryB12E+j3OIFd+Qbohw13j3YLyKSN8Kv76JZ1FGnLYr7vuQW4Q9xImBt9lr1LLFn+STkQGsZFXE63WUzaUv1pHeRxDGFhF6/il6f0PrLyZ0A4jIeC8KAxlOBGUgIABap7wCWshA8H5Ly05fpQktcoaMYpgUMaeMXMZFOs/COS3fZRUg7Cfo0v5FXM9lDDmc2ziiNIiMpPE2/MJpyoh3WCUUROPqDbrc8m/Ub76JHkNOxSX5ZEUFQaZ1RZUs3pF2gv8Jv77LHsOEELaoN/8NGyn+IORL3wOC00S8ep8WBJjInBXd0jHj3cObMC/8e0joEo2wLPPCshH7xzOrfywH7kGkmyeKL5vaZJsiXVym8wMHUqr/3EZ5ISW1RUWnfHMbJ3l1h2ZOr5Yp4e376GuYLajChkZehDl1xGkym3zyn9uwgL7K+BqvaOUQcT+gwskgSmR8T7cTKD8gbxX5K+ByRmcxCP51R4j2PcJw2gVd0fHwltgT+c9LqN8lN+/pzWty0zHJ5SsoZb2YfJj85pH/XNIVCxOrxAq6sGz47z1dgEGxLNSBZQuswKphBQzLZViwYPueZfYJmOdaHViYY2EXUSx44Yr955aJ6yW85jE0RppnuxQNCJTRHIFGtIGiIYaGGJrF0Vg74fVrVusFK86bWG6JZUlYDsNCDMu2GZbVi+UJLJ+30qJYFqfLo1iYIdm9SH6JZLcgYUYV0wmogehBCjiS4zDtQh5DYpx3GE2sdbZEk9NEQqaAEkLsgXL6oVAJhVugXKsO5fZDCZV3PKbyoD1XTNsJlG/WWAUSqKBwHQrq1ZHQeMfHdSjOdQ8zXWACtIN+LKHxTsCYZTGNB7kDlsPs2m7BcnawnBLLkbA81kTRR1AsjPqx3BLLk7AYuwLGLjBIwLL6sYS+u4hpqcW11KnThTld3cpFsYTGuxazQ2DNFYMELNbfOKy7wbif90EHFmZYAcXylbAsofSuzeWI63IMuE4gDiapvbcDJtTexazjspw6GDIdrmFMXbGk+W4dDeqKLKsDjTUTWZw2PnZgrx/NLtFchubWhYlsjub4u2g22kET6u/C828ZM2q0OawHsxBTDsfsR3M60DhtLtMOK7B30fAumtuPBl3uSy4NBbTSCDyuIIGM5nMp8JaifrTSDDxGmy11Gxbi45GFW9DMHbTSEHxTQmO0WRZvKSzpBbS6iWLJFBA4T7bZAedxON6pASNU4Epr4I6PzYZem3W3lokkf8XBA3BWBxzmcNxlcXELXLALV9pDwOzBtiQ47gFhk1NXt3zH3IUTBuGZrE+ybamxFu8sHa503gB1TgnHRj0bS3CYSdYGp6YJ56BdOHcAjjeWjwuDcMImPJfDcQtzGRy3fhzgFjh/F87vgOOi4AOz4zDzd80BuKADjlsFt3/XxLtwruRfISiLx8IqPI8ril83WdvmcBwNDaAJo/C4143Numdk27w7sZmeuPYAnNUBh2Q47LfAebtwwih8G9fhuFHY3JG3LKZ2rjsAJ4zC5yMY5ibL9MTmfq7Ne2K3riee3Fio88VOCWdLcL4EVzbWr8P5u3DCKHw+TmBmspiLwuGNtVnv6dX1xPN24YRR+EE7HOs9bQcNwsEOEVgYRYA477AEZzMvB/kczh6AE0YROEyLMXMmbD4PsnhjuRrLcJIzQfeDcMwSLpDgPA7HqKMiuWfucC+csIqAO/nYlRrLqUPcp5PhmGRhyCrhhFUEAeedJymKxf0Jl3V3njcAZ5dwdh0Oc6vgw6zLiWtROwkN96O5whHDg3CwwYcjjAKZ3EvHgdRYj08fMLNZH+0qioTnquHZtiKeV+E5PXiYuzy+NYDnV3hBD57Du3cfD+AFHXi8C7U8i0vX3MXzrR081yzxLNYPOGZ9wor41NDnwRDf7cZzAQ914PERA3ncNhCfIPreAKBVAto8IMVDIh733fksxUY8KBUMANodgJyDyHG5RpueGiAuAfm40QD0Pe5zm8zkAtQNSGNAlY3wyY9oss+dPe4oOx5zMQJnANBVBPS5Fg4CVlbiMKtzLEkoPusVLNPmgP4AoN8FiGVAykwVwGAA0PN4P8M7rqBHyjBZ9kxFQOqz0iBkj5gpIlJEDPgkdxjRUkMkAxQPbZr2AKKtiIgdEXrt6XBgEuhV1uJya2GOB7JkRI+7WQ1EtItYmYvLtRu3IgYu4ojuAKKrhogtHjFDpjeAWBmM14uIfdHqoAURUUS6ls3zFRE97PUgmjXEoAvRlBAdU2g4Qv2IvqmIKOboCFkDiEgR0eEhchkxcCREcPr9ymZ8NuA7joTosr7R8bBotT2AaA8g8vHACTxu18gZQKxsxvd7EN3SrpE7gOgoImLbVUSsbIZ7sQ1EHpt0XVcg1rtwZJoSJExRfE8R0g8EZDAE6atBeshqh/R2ISuzEZmpdkhbdJGWOQAZVHbDw+IypMUnGJ7jC0hrCBL1QwrD8Xw+HvZDwhwtqCwnCHogfcTn30jKMrVClqZDx7puSDswOSQegsQVpN0H6YpR1nKGIJ0K0q1DNl1mi/cYljuEWBoPQiyS4bAZpcutkYUKAssUzfYkRFdChGlb4HUglgbOVCjAPB6vAOlXkEEfpOcIKoMhyNJ2EM9nNCAdNjsKAo8rum22QaIKklj9AGbA3XGThq9roFYnnUQbVUExdpVBrQrU7gV1fUEp6gF1KGhlQrbfCxogQak9CIoVQRFykDJoZUaYd0kstOA1MG3Rvdt4ENNVxHSxrYxZmZKDdjEtkdhCvuiUeI64D9PvxRSmhMh8x1LAdClmZUwO7sPEHu/ieQK6BxOZipieGNUVMCtb4sG9dkzbFD2y7Q9iVqbkcrmzsLdIe7micEL0TVg2T6cF01bEdIXOK2BWhsSj/R2Ygcv1E6MeTFa8UNkRz/m1Y2IRLUU8kd6HWdmRh1swuVOMsIMEP+1BTE8R0xOOkgKmr4ZJ5iyWMmZlR2KO0Y5p8cg94nn2HkyrsiO/F1MkoRqYyJIGOoZZ2ZHw4euYlmmJKBHP4iHstmFyOmmpgmWpYbpmKfdhTFsR03IFnf4gJlbEdCxTGdNRxPQ8dTrdfkxRauaZtjqdlR2JOUw7pi3mwI45iOkrYrpiPuigQcygF7Psl7zA9hQwA1oSZaph+pbtK2NWdsRTHB2YDk+tIccexLQ6MJGICYuIqy8mCQ4eBLUVQQPqoCuCYlVQbKqDVqYkpoZBfYbkcnUKXF5bhBy5u7NbMN1+zIBnGAORqqDeRRembVJMTwnTMpEI/jjeIKaviIlNpIwZKGK6jjB5fwgTm4qYgQjEKmCWpmSZEqYnYyLaZooZtGGiOqaliFnqp2sO0ml3YPI6W8sNRB8qGs9BPdJd9AGX1kQrct6yeg0AFuUXZRVcIIDRDjCWgBEFdjqAGcEYC4I9nuhUxXVVCbadBsHkJ78H2FMFdoVL7lqDwAS5gMUk83B1+XrCdj2GxaDkU1m0iUJYWAerQ3OxLhV20SN052yhaaOkHzhT/mN6zX8+MfD8dbm1NXynvgaVWIePOA3SJz9Orld0G7qLyR+w3CJLk3gOyy/ebQr2x10RhWvjlu+kfjlPtzn/5Spdz+BCWkxzvc3ZjZt0Fq/I5ykX6Pby6Szlf97ECdt2730Yr2CTbvoh8je7+8diOy+XjoQ5bEZEvgwU0y2N2G51dGXEi3AFm08z1HqDici2ySb9uqBUX0cZLBf5V8SWj8ASCthAlH9tnibpWjSp2oKWfzOGFTFse6XLAp6DBd58o/kw+cwwYAfUebpeb5NyvRrlzwY2HKQrTOA8Rdh51rgNM7ook560OQvnn4338YKhAEPpCs3LbE0BltGatI6dV5QsCK9huSI/mZVyrIhWsF4EHn5BRPcvtpk/QwPBwYbh5C+2s95DnfbbZbxK83RDV5UATfEciFt+z0vhbrYFY/3vWbgODdgK6jaNGDnlRqtiZTQByeKErhe9pRoIi44K0oCQS+cl7BpJCY3TVfpIF96k888CLyScTph8xTaCrBkggzASrzAljedsH/7fvyfhmq2XeR8uQnoSQQorSeDHy0fyHF/GwjbNrTHnffzAlQ04wsHfhl/ix7Cp0LBei62QW6WzOvuvwlWUsK+Sr8cp0yWOdRPmacKPMYgiethnydv7jC5Xpa1bfAkJ+xaGpMlXy/hLuMr4UQPGhyVDuEziNVHdV9t8RoTBjhLIYq7GdAGacQdr5Kk4qJqvthnjdPSQxY9RJki/hZU57FuXORh9yk9sSmAB2DIRrbhdhtmaaO6W9mR0dS3tQu5T0YnQLqRmeJ9oL7PepHDWwT+SGFYO0eV0bEOAdzPYT5evkm9dU1V/jq+EowuUgJxyz0pWYrml8LA08H9Zzqf/adwVpF94LJbGf3xdRokBW/jOQd7/afwHLCT6KJSWLXfO/5PuOlvAiiq+3Am2wkwTYk/EQsnYasNsl7ATVhzV1jmxwz52tqt+qm8Jerkm+jt/xp6gT2JrTwvq66SNPZ92tuLcc2vMJ3kxYb5Mt6vFu02UMAbfxsQcM7HrZWMdGDYhO7PfOrCewfBPPSD9ewj69xD07yFo7CHoqbVPrs6guaWLqOvbMvMfwJu+IxpT3MfQRyOPTMHdgMxEAoiTLeKHByosWH9+F8FB8cDWR9Kfv4bDVmazMAq8mT2due58ii3kT2e+g6bBPAgecBQFluXx54kuk39TzSVdXAfwOvzG3XsI+sL2ky/CbBYS84J7rA8WC3Vhcx268pr8Yopl4nxLBLq8tnFUTMs6cjpo1Z4VAxQwLgemEBH+kYXV3jVEUpOS0wBFF++y8QD2+iBiWLLFuIyndB27yXZLhgXvwLQAwjtw2stNuAFWrMONzJs5OwECjnf437D5PwyFE/wrsn51p5uwmC+R8QvdbtxAJvL+QgiCEwhA49lXBaMIsNhyCDb4vpxtyTP0qCfyyx3pdeFjy4htuA1zZ37yzX2cgI7BOm16XAKh/2u8gFN2rNqRzPdk0DPWbN0+6Gse0QNxqP7YnuXSugGiw7B/QTwnZgT2JEgrmBTY9quwzSrw4w3d5pqIb0ZGk/soy8IYWv4yI501eRsWRZMhgciE7jVtVv7Ci+8ghcq74CL/VAoDGgo1FB8rbXhLhZVVRxPdEWOGBfEw0MHG15KxFIya6wh2g2VCfwXHYZBPxLVNn2pTzWS7WnHf7D6VNsmmWzIdoonUk8nj+m4MqDxpRCM6aR3PjigapCNxZEq7QMZozQ34hfDnT7FZP/7uEqg1fd8YfWdrYu2034RZvhyLeDJ6x9R1fTblYf6CHqsA88P3cJaM6IF722SV50QdWix72iBddTPUhVjlkW6adyG1ntoayVL315jaiOmodtOdPD5Hoz6S5o8hIahe7ZXQM2Xzt23yyPZ80lY61Q7vZzNo0HLOIUOaHsHhOZRAzsRaoD55UAw6Dhe0eFdttDuo/owghG5n4wjm/HuUR/TU4SO5UbbY9LC1ZderNF0YG8ZTzTqsFnWL658Wh7U0Wlye2XkSb+DnxUsPH3u2m9whXy3cZIWRx+oXmj5q2e01aywBWois5kUfVARjDKCwm8Dz+wUhoFqs7WVE8yin6CTG4ANd+zHkd5+qFxyngW2SbnfDD+owjaK1w6MZPzysvXfRvoF0fZNaLz/V3T+kW4LsJ60jOMBHnA+enxNVtc8dHic6ZHgGXghdfT0ovX5dPHjoZQwp0nL9wfmyfYR+ZSTB4b3cl1KapwpyjiJL1Ka0Km0XArYa2dJ49YVm7M+LI/vN4/nhBfh48/gxZN4271CROGstPjtt33MSb5+lcPc1Z7sSbmXMl6vtOk62MCCLu5drmO+yXZvPSw0UAjZYoyQb3jfWZp3UTMfypRRmLiLI01DaqxCKxrpGoNOGfBQcJ9QvPl3Uk+51MBTqmJ65Fu4d11LvR20N9dRRML5agFVvPd3Xt5/WYq+1ziXdbE7m3h4+2H/YoN1IVonagqzPiARpH7fDww3sc+K659xnEGG2hmcnXSGiswnz7dslNUYVWCFx5BFllOnn3uarvVMUKGRH8LMjgJpNn34iiovPJyaIhqdfRGOqU/BapdyIjf3+PerMbuoiXLqP1+DMCx/BWxilNcPhkHZVxYdPvB8mkoErv/wcC8zZyahTp6du42JyTT6dF/t3iWGcscOe6ZIwttztpnyE+P0xrMaCfUXX/O5NtE6zmK/NbtBcHuZ60f3Tp3J1VOMw14sJwX2Mk3C1o2Np8wZpxjoV66cn/+cW9t5gy6s7m4DPvwnO+TfBPf8meGfRBAI9vlMnT0jetwwhsODUmEd0LwHCJ77SGroYeH+Hqb5gqnUIpnJuGlfZNs4pQT/HVfnuR1jXHmfRHI7hhgPYn3Y7cUFUx9gJN2F5LiwhLXcQOPmYumeSCA+PV3qFNH8u+efIDu0/sjCJn5ce0tdzsv98orwK15uuKFFDlNfb7FSVlwdN6R1BnGMMMm0R2sZsM8z5zhitkduGOP/Mi9aPINK9M3xmW5Via+qkc3zURSWHIwFCGNrHcgalcmh5jBU4Hg5nvCEfWYfZ591+op4t17ydbpvmPaMy+xzauHdisra46GyTPG21A0pJ2YE1SWfQ9tZe9RnJvVNVm46yLH1/L2dqt0XUz9/NmeyrDvVcSr0+IVy1TMdgqyatnY29O4TTptPGjy+pj3UNj//DkrTxkNmlfX3iNtEmId0X7UOaLCihxJaShXET5WGLiLX3lAOFPWecw1cnnDa36xwhtztKgfS+LXQOXzw0SsrTUpiCCmlp35q9y4Gcc03gOuMmcI+zYORdvDK+RjQ52yqHRuUSefzPETN2WivQxVB8jgl4l7XIPXyMjbiqefFcZ0JhzwTWAldPp2HPNJPTK5Vxl4geSi7O4eUy3q57uGL46Xg6tIlbOQs8KFMPmm07KYM9hXJN+1wZbOndkdMzHtUWJXQyv6Vdl0Wqe2ISnV7zu3JSY9E0Tpx0yDoFFw9mmCOwdkSmjpVDGk41s93Xc2OZbvOWmFstDauhh1VP+/1EuBm1ZdXpESB5y9TlkGsuxoozD8u9N9905ks5h8v5uzOn04El1Ro1c1jle5NLjdWOJ6rvGmsN+c+tJDuHLYcUerjBRSotO7h8HjePoFhvH6iVw07F2pqWetjqt6ogVpwLdJxKaci0/Qlagf4UrVAssNa8FYrrUE7fip3y+4N05CNU4Huiy/EPy9q77Yx4FGzA/WkV6fzwCersj5Py7/bIjpFsPGlZw0C6WMeIjqsSm9ddYGh4l9GbMNk+hPSgtxbB6R4pVAu8uWdSrbD3ZhHumVQrDO8oXUpL+1w+slT2DXXPZPdohThae9/unulZCApZ49OvOn/WLkG9Pbl7+JjPQWUx1l6Gh03Tn8Gqi24WH3aXgmMcseOxhngj6cq7eRQ+uz60dTFAg1BGp6fnmLBnTYerAe9tZUM9KO/vt8mCHqB8iOKs0zJYYes9R2flViiS8TQtMXCV62cOyvuD1s94cuz+dTKPEpqz1ST/XaaWxrLC8Uq8rHG7h3GYZY3KrIMWsJyyX+0qYBmLplFif0Nja2m/etev6MRTPJzwbA/41IqutGK30qEx06o6q75jR5wvJ3r7DHuXI9Q278i0Po5gP8dEZYWOdS7zYwUh9+1D01w88Xt0sv219xJma5fU7q2NHExSK6tAihvolRtHtuQ4q9+qHOffItgCjMx4R9hjbCABrrh/nsYtUNw+T+MWKGbKNW6BYnnRaVtwqBIKuZO6+vkSClewVXGL0eey9SrMCNT42xfepFm4at+3sLZ17rMKKnQclRRiqtMj7HF8Sh9yaInzedQt713EeoTl6ntppsry9GNk5/dz9hRCitMjFE/sFYt2FTxV7eIRE1+hAEK/MIraIaneeZTZtNZtyJ3Q37dkvG6rh/Jatym6K9LkoEe0jRON9co+VJfBsPVoLqWKQq+12L9jv6hDrmcaUzQaFZx0Z9fHInK/Nc4Kx3l4hy91OSiDO45oPbkT3nqOVyvvDxrmO0x23WsN0d6E2YyuJdBSHsO+dJwv4+TReJGGRb7biXqniMMfVHx6rUKf+owqv92gX89H2+RkLwfaUfD+WQt83fzRbg3wT1ke07rRb6vRHZSjhzExt2Kw0ppa+5SCUNgKxT1XQTitguCJptOxXCEQ4pwry3FvRz6mJzBO+QguGa1XkZJ9+k66q6JrLJoOVqVtnZkOWhqydeKrRDxRawd7FRbF6qBRnjFLx8bi+35BzuFlh31TkzbmZ+FMc9afmulKO+XWih11d+jHLYEcqf8YXoU1cLDRtDHdPskeTCNJqNrPTpOY2mS43xnYDKx2appW3G8Pvj0r2jNtPSjtNgqz1UGPcT5aWV73rs24tTTvBNs27zWzal2a2z5DOWjQey+9VUgb1rYI14t2ldVO04Gd2Q+9qefBGnaMhev79YQKi/ymR9hDYL/8lfLqUA3LRVSInx6hCGCcedJpnXVbZaHEQSNle41FyFJIAvqHV4FxEzRIq7SRP2qZxn7qiVXWdPiaDhFdLD1l5UI3+/Siyi/9SH2j8f5wncTY+1XsN14qGJF/rgUTfmvBxOuM7szaEszQMGY98ZQz4NqttOyWyFjxmL04qlLe6h9+k4JxOarX0vBpwKgK9KwqUfIQWROC81EBbZmtMJP3z4bZXsXs07HUVi7iOQeWulqwVHmRzDmw1NGBpVhBS53Ds3T0I8pOydWuOpqxaBpJ/2wdOWXryClLR05ZOnIK6cgpNCqnxinx0IlF2qlRrXBKExZVun3icL9CQXJtBzetgv6dHesp95pTqjeuncqpO0ergzlPt5RdwaXEA/Uxh05371earlL0iQ8fiN0vV69eQaFf9Yenstq9dqa6Tkb6jN2ED1u7sl9oWyW/PLB3/+mIV+jb6wvAtdJ53Mb5RqXaw0O+JIDG7SosHtLWDU+qxfrDJWuaya8rBDlann2kMbeqU9CHKI2mF9/0mu2UaYNTVmtMkMqerQOFuEHrJiIfluEqOmgl7phCGKs8ZdycmF5UBTpWnQTjVp2MyalTnvHSzamxqBqTU46WnGrUgvCdJHTjnWZ7RASnr9ro4pReZwkITulVjRGMGlwcdzTUc9+RtgityT0i80wo5gSfDb2ly3kuBAcVfzXqgfwz46KvJRe9Ghf14ZWnJa9cLXnlaskrR0teOVryCmvJK6wlr2wteWVryStLS15Z4/Jq1LoE3XiFtOOVbkzSjD1V3Y1ewfrpuOVbYx5IqFuF0rhbZe1ZoaSwuLt+SqfeSYrpuMd07lehpMBRfD4VSvjk0dOJ1VaR8azcm9MaqT6X1Buf4owV/d+vNkOhSsw9Qj3SSBx1R8087cdRhdrQ2t77unN03KMM9lwdo1C6dYzdC0YtVNExua1bSc+4FSJjJQu0IkeruVQtl6IVUVVKSiuyOFV69QQlr/Qii1OlV6FLySu9yOJU6VXqUvJKL7I4VZqV4AheOWeSmuT06lX2UnLRPS8ualaoI7ioV6UOp+rsimL0ilVyqvTcD6WHi3uGoWHeSH8hLcdkZvslynKOPs9Iw6PFh7hYwmWyXc9gAolMOB69iL5BNAr/iqxf3ekmLOZLNCFY4in89PT/AEYvIMd8hQEA
```

</details>

### Operating System

Android

### Additional Information

- It works with Carriers, Missile Cruisers and Nuclear Submarines, allowing you to infinitely stack any unit they can carry (any aircraft except Stealth Bomber for Carriers ; Guided and Nuclear Missiles for Missile Cruisers and Nuclear Submarines).

- Upon reaching 6 air units, you can build more air units, you can move more air units into the city, but you can't buy more air units.

- Upon loading the attached save game, I sometimes encounter #10375 (it seems random, I can't find a pattern). You can check this by opening a city screen and looking under "Units" if the icons for Privateer, Machine Gun and Marine are missing.

**Comment by yairm210** [comment]
I tried 5 times to open/close the game to recreate the missing icons, no luck :(
Thanks for the bug report!!

**Comment by Anon177013** [comment]
No problem, thank you for this awesome game and for fixing things so quickly!

Yeah that one's pretty tricky, it occasionally happens and I don't know what triggers it. I found a way to reproduce the behavior (the missing icons) though. I wrote it [here](https://github.com/yairm210/Unciv/issues/10375#issuecomment-2201275138), along with other infos. 

If it happens again, is there a way to send you more useful data about why/how it happened?
